js/app.js

```diff
@@ -0,0 +1,43 @@
+const totalZombies = document.querySelector('.total-zombies'),
+      deadZombies = document.querySelector('.dead-zombies'),
+      smallZombies = document.querySelector('.small-zombies'),
+      madZombies = document.querySelector('.mad-zombies'),
+      strongZombies = document.querySelector('.strong-zombies');
+
+let totalZombiesCount = 0,
+    deadZombiesCount = 0,
+    smallZombiesCount = 0,
+    madZombiesCount = 0,
+    strongZombiesCount = 0;
+
+countZombieType = (zombies) => {
```

Comment:
is there a chance to implement this task without any function?

Reply:
Yes, of course.